README.md

```diff
@@ -1 +1,69 @@
-# My-Portfolio-Project
\ No newline at end of file
+📖 MY PORTFOLIO PROJECT
+This is My Portfolio Project dashboard.
+
+My Portfolio Project dashboard shows my skills as a software developer.
+
+🛠 Built With
+Tech Stack
+
+Client
+HTML and CSS
+
+🚀 Live Demo
+Add a link to your deployed project.
+
+https://github.com/AbednegoKibitok/My-Portfolio-Project
+
+💻 Getting Started
+Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+Prerequisites
+In order to run this project you need: HTML, CSS, Code editor and a Browser.
+
+Setup
+Clone this repository to your desired folder: https://github.com/AbednegoKibitok/My-Portfolio-Project
+
+Deployment
+You can deploy this project using: Visual Studio Code
+
+👥 Authors
+Mention all of the collaborators of this project.
+
+👤 Author1
+
+GitHub: @AbednegoKibitok
+Twitter: @AbednegoKibitok
+LinkedIn: Abednego Kibitok
+
+🔭 Future Features
+Describe 1 - 3 features you will add to the project.
+
+My projects
+Languages that I am proficient in.
+My frameworks
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+(back to top)
+
+⭐️ Show your support
+Write a message to encourage readers to support your project
+
+If you like this project, contribute to it via my github page.
+
+(back to top)
+
+🙏 Acknowledgments
+Give credit to everyone who inspired your codebase.
+
+I would like to thank Microverse community for such a great support.
+
+(back to top)
+
+📝 License
+This project is MIT licensed.
```

Comment:
- [ ] Then, please ensure to create a license file in your root folder and add the link to your README file. 👍 We add a README file to a repository to communicate important information about project. A README, along with a repository license, communicates expectations for your project and helps you manage contributions.